src/constants/oracleConfig.ts

```diff
@@ -1,40 +1,52 @@
 import { oracle } from "@uma/sdk";
 import { CHAINS, ChainId } from "./blockchain";
 
+// metamask local chain for testing
+const mmChainId = 1337;
```

Comment:
nit: any reason this can't be `ChainId.MetaMask` ?

Reply:
i can change that